packages/terra-clinical-result-view/src/ResultView.jsx

```diff
@@ -0,0 +1,195 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import IconDocuments from 'terra-icon/lib/icon/IconDocuments';
+import IconComment from 'terra-icon/lib/icon/IconComment';
+import IconModified from 'terra-icon/lib/icon/IconModified';
+import IconUnexpected from 'terra-icon/lib/icon/IconUnexpected';
+
+import IconAbnormal from 'terra-icon/lib/icon/IconAbnormal';
+import IconAlert from 'terra-icon/lib/icon/IconAlert';
+import IconHigh from 'terra-icon/lib/icon/IconHigh';
+import IconLow from 'terra-icon/lib/icon/IconLow';
+import styles from './ResultView.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   *  An array of one or more results.
+   */
+  results: PropTypes.array,
+  /**
+   *  The age display of the result. Does not handle formatting.
+   */
+  timeDisplay: PropTypes.string,
+  /**
+   *  Does the result have related documents. Adds an indicator.
+   */
+  hasDocument: PropTypes.bool,
+  /**
+   *  Has the result been modified. Adds an indicator.
+   */
+  isModified: PropTypes.bool,
+  /**
+   *  Are there comments related to the result. Adds an indicator.
+   */
+  hasComment: PropTypes.bool,
+  /**
+   *  Is the view aligned to the right. Defaults to false.
+   */
+  alignEnd: PropTypes.bool,
+  /**
+   *  Can the result text be truncated. Defaults to true.
+   */
+  isTruncated: PropTypes.bool,
+  /**
+   *  Should the view have padding. Defaults to false.
+   */
+  isPadded: PropTypes.bool,
+};
+
+const defaultProps = {
+  results: [],
+  timeDisplay: null,
+  hasDocument: false,
+  hasComment: false,
+  isModified: false,
+  alignEnd: false,
+  isTruncated: true,
+  isPadded: false,
+};
+
+function documentsDiv() {
+  return <IconDocuments />;
+}
+
+const resultIndicator = (result) => {
+  let icon = null;
+
+  if (!result.normalcy || !result.normalcy.length) {
+    return null;
+  }
+
+  if (result.normalcy === 'ABNORMAL') {
+    icon = <IconAbnormal />;
```

Comment:
We can define/use constants for the normalcy, then export them like [this](https://github.com/cerner/terra-core/blob/master/packages/terra-avatar/src/Avatar.jsx#L101) for consumers to use.